apps/client/src/components/PhoneNumber.tsx

```diff
@@ -19,7 +19,7 @@ export type Props = {
 
 export default ({
   darkBackground = false,
-  eventName,
+  eventLocation = sections.FOOTER,
```

Comment:
Why is Footer the default section to call this event? I would either remove the default and make it required or remove the default and only pass an eventName if `eventLocation` has been set.